nilearn/_utils/data_gen.py

```diff
@@ -1241,9 +1272,34 @@ def _init_fields(subject, session, task, run):
     return fields
 
 
+def _write_bids_raw_anat(subses_dir: Path, subject: str, session: str) -> None:
+    """Create a dummy anat T1w file.
+
+    Parameters
+    ----------
+    subses_dir : :obj:`Path`
+        Subject session directory
+
+    subject : :obj:`str`
+        Subject label
+
+    session : :obj:`str`
+        Session label
+    """
+    anat_path = subses_dir / "anat"
+    anat_path.mkdir(parents=True, exist_ok=True)
+    fields = {
+        "suffix": "T1w",
+        "extension": "nii.gz",
+        "entities": {"sub": subject, "ses": session},
+    }
+    anat_file = anat_path / _create_bids_filename(fields)
+    open(anat_file, "w")
```

Comment:
```suggestion
    (anat_path / _create_bids_filename(fields)).write_text("")
```
Also, `open()` should never be used without `with`.